UnitySetup/UnitySetup.psm1

```diff
@@ -367,9 +387,23 @@ function Find-UnitySetupInstaller {
         [UnitySetupComponent] $Components = [UnitySetupComponent]::All,
 
         [parameter(Mandatory = $false)]
-        [string] $Hash = ""
+        [string] $Hash = "",
+
+        [parameter(Mandatory = $false)]
+        [string]$Cache = [io.Path]::Combine("~", ".unitysetup")
```

Comment:
Would prefer PS native `Join-Path` over `[io.Path]::Combine`.

Reply:
It seems both ways are widely used throughout the file. This one I copied from https://github.com/microsoft/unitysetup.powershell/blob/d2cbd09cdae4ab84db5f798e6f2f421c69f893ce/UnitySetup/UnitySetup.psm1#L973

Will try to replace all entries to `Join-Path`